README.md

```diff
@@ -2,8 +2,7 @@
 These are dev tools to play with [3scale/porta](https://github.com/3scale/porta) in local environment. They include:
 
 1. a CLI to ease commands such as starting the Rails server, launch dependencies, deploy to OpenShift, etc (see the [full list](#supported-commands) of commands)
-2. [Porxy](porxy/README.md), a porksy proxy to porta
-
+2. [Porxy](porxy/README.md), a porksy reverse proxy to porta. Because Apisonator has it's own DNS resolution that we can't override with `/etc/hosts`, we need this reverse proxy.
```

Comment:
```suggestion
2. [Porxy](porxy/README.md), a porksy reverse proxy to porta. Because APIcast has it's own DNS resolution that we can't override with `/etc/hosts`, we need this reverse proxy.
```

Reply:
Thank you, good catch!